src/pages/components/link/usage.mdx

```diff
@@ -28,14 +46,105 @@ tabs: ["Usage", "Style", "Code", "Accessibility"]
   `}</ComponentVariant>
 </ComponentDemo>
 
-## General guidance
+## Overview
+
+Links are used as navigational elements and can be used on their own or inline with text. Similar to other interactive elements, too many links will clutter a page and make it difficult for users to identify their next steps. This is especially true for inline links, which should be used sparingly.
+
+### When to use
+
+Use links when you want users to:
+
+- Navigate to a different page within the application
+- Navigate to an entirely different site
+- Jump to a elements on the same page
+- Link to emails or phone numbers
+
+### When not to use
+
+Use a button instead of a link for actions that will change data or manipulate how it is displayed, change a state, or trigger an action. Buttons should never be used for navigational actions.
+
+## Formatting
 
-_Links_ are used primarily as a navigational element. Links may also change what or how data is displayed (i.e., view more, show all). If the action taken by the user will change or manipulate data, use a [button](/components/button/usage).
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![link anatomy image](images/01_link_anatomy_608.png)
+
+</Column>
+</Row>
+
+1. **Label:** Communicates what is being linked to.
+
+### Alignment
+
+#### Links with icons
+
+The default link component can be paired with an icon. Use 16px icons and place them to the right of the link. Icons should always be the same color as the link text.
+
+<Row>
+<Column colLg={8}>
+
+![link with icon example](images/02_link_icon_608.png)
+
+</Column>
+</Row>
 
 ## Content
 
-- Use text for links rather than graphics or icons.
-- Links should be three words or less.
-- Text should be consistent with the title of the intended destination.
-- Use caution with links that are several words long. It is recommended that links are long enough to be understood by the user, but short enough to prevent text wrapping.
-- Avoid the term “click here,” other links to “here,” or the web address itself. Instead, use a meaningful descriptive label for the link, and match the destination site name.
+We recommend links be three words or fewer and they should be consistent with the title of the intended destination. Links need to be clear enough to be understood by the user, but should not be so long that the text wraps unless used inline. Because links take users to a new location, it is important that their labels accurately reflect the content users will find at the link destination. Use meaningful labels for links and avoid terms like "click here" or the web address itself.
```

Comment:
```suggestion
We recommend links be three words or fewer. Because links take users to a new location, it is important that their labels accurately reflect the content users will find at the link destination. Use meaningful labels for links and avoid terms like "click here" or the web address itself.

Links need to be clear enough to be understood by the user, but should not be so long that the text wraps unless used inline. 
```